src/navbar/navbar-action-buttons.styles.tsx

```diff
@@ -17,6 +17,17 @@ export const Wrapper = styled.ul`
     }
 `;
 
+export const UncollapsableWrapper = styled.ul`
```

Comment:
nitpicky, but we can keep the spelling consistent for all instances of collapsible
```suggestion
export const UncollapsibleWrapper = styled.ul`
```